common/source/docs/common-electro-permanent-magnet-V3.rst

```diff
@@ -57,7 +57,7 @@ Firmware setup
  
 .. note::
 
-    A pull request has been issued to include the changes in the next APM release. This documentation will be update when the request has been accepted. If this annoys you please write a comment `here <https://github.com/ArduPilot/ardupilot/pull/4361>`__
+    The firmware update is in maser and do, to be released with Copter-3.4A.
```

Comment:
Spelling errors. Should be 

    The firmware update is in master and is to be released with Copter-3.4A.